ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -227,7 +239,6 @@ public synchronized void setText(String text, boolean animate) {
         }
 
         columnManager.setText(targetText, animate);
-        columnManager.setAnimationProgress(animate ? 0f : 1f);
```

Comment:
We're now not calling `setAnimationProgress(0)` at the beginning of the animation. I assume this is fine since the animator itself should start by calling 0, but just want to double check that there's nothing missing.